packages/mobile/src/screens/search-screen/SearchScreen.tsx

```diff
@@ -57,21 +57,30 @@ export const SearchScreen = () => {
     if (searchQuery && hasResults) {
       return <SearchResults />
     }
-    if (searchQuery && searchResultQuery && !hasResults) {
+    if (
+      searchQuery &&
+      searchResultQuery &&
+      searchQuery === searchResultQuery &&
+      !hasResults
+    ) {
       return <EmptySearch query={searchResultQuery} />
     }
     return <SearchHistory />
   }
 
+  const renderSearchBar = () => {
```

Comment:
nit but can just be `searchBar` not a fn

Reply:
was just following the pattern. let's chat at next client syncobinko